stock_operating_unit/migrations/15.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Jarsa
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+
+def migrate(cr, installed_version):
+    cr.execute(
+        "ALTER TABLE stock_quant ADD COLUMN IF NOT EXISTS operating_unit_id int4;"
+    )
+    cr.execute(
+        """
+        UPDATE stock_quant sq
+        SET sq.operating_unit_id = sl.operating_unit_id
```

Comment:
```suggestion
        SET operating_unit_id = sl.operating_unit_id
```
Table name should not be included in the target column of the SET clause